pkg/ingester/ingester_test.go

```diff
@@ -833,8 +833,8 @@ func TestIngester_GetDetectedLabels(t *testing.T) {
 	require.NoError(t, err)
 
 	res, err := i.GetDetectedLabels(ctx, &logproto.DetectedLabelsRequest{
-		Start: &[]time.Time{time.Now().Add(11 * time.Nanosecond)}[0],
-		End:   nil,
+		Start: []time.Time{time.Now().Add(11 * time.Nanosecond)}[0],
+		End:   []time.Time{time.Now().Add(11 * time.Nanosecond)}[0],
```

Comment:
Why the same time?

Reply:
will change this, missed updating it after changing to value semantics